### DISCUSSION TITLE
Gross value in 'Overview' dashboard shows wrong value.

### DISCUSSION THREAD

**Discussion by dennisbos84** [teslamate-org/teslamate#1304] [Q&A]
**Description**
The gross consumption value as reported in the 'overview' dashboard is way off. Like ten times too much.

**Expected behavior**
A proper value for gross consumption.

**Screenshots**
<img width="602" src="https://user-images.githubusercontent.com/19466520/105688283-b155f400-5ef9-11eb-8126-8a9bb9dd3acc.png">

**Environment**
- TeslaMate version: 1.21.1
- Type of installation: Docker
- OS TeslaMate is installed on: CentOS
- User OS & Browser: macOS 11.1 + Chrome 90.0.4398.0

**Comment by adriankumpf** [comment]
What time period did you set in Grafana and how much did you actually drive during that time versus how long was the car parked (and online)?
